### ISSUE TITLE
beta.etherpad.org is offline

### ISSUE THREAD

**Issue by stenioaraujo** [ether/etherpad-lite#3203]

**Comment by contra** [comment]
Yeah, it's been down for almost a week now and I don't think anyone has noticed. 504.

**Comment by webzwo0i** [comment]
Thanks for reporting. There was some memory activity (mmap&munmap). Hopefully someone finds a way to reproduce this. I updated to v6.11.0.
However, please note that this is a test instance. Please don't use it for any serious data and do the backups yourself. AFAIK there are no backups and never will be.
